### PR TITLE
[15.0][IMP] mrp_project_analytic: Add mark_analytic_mrp_from_child

### PR DESCRIPTION
**Pull request by Anxo82**
Add mark_analytic_mrp_from_child for to mark  the new field analytic_mrp_from_child to true the journal items from child productions

### CODE REVIEW

**Comment by Anxo82** [comment]
Review please

File: mrp_project_analytic/views/account_move_line_view.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="account_move_line_form_view" model="ir.ui.view">
+        <field name="name">account.move.line.form</field>
+        <field name="model">account.move.line</field>
+        <field name="inherit_id" ref="account.view_move_line_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='analytic_tag_ids']" position="after">
+                <field name="analytic_mrp_from_child" widget="boolean_toggle"/>
```
Comment (dalonsod) [inline mrp_project_analytic/views/account_move_line_view.xml]: Add control access:
```suggestion
                <field name="analytic_mrp_from_child" widget="boolean_toggle" groups="analytic.group_analytic_accounting"/>
```

like

https://github.com/odoo/odoo/blob/c44aece8c91dffd943f66293cee629c33207b241/addons/account/views/account_move_views.xml#L72C75-L72C118

**Review by dalonsod — CHANGES_REQUESTED** [review]
Only code review :+1: 

Only cosmetic changes before I test it, see comment. And also, add in `DESCRIPTON.rst` a breif description for added functionality (in a more human readable way than commit title)

**Comment by Anxo82** [comment]
Review please

File: mrp_project_analytic/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -1,2 +1,4 @@
 For those MRPs linked to projects, when account moves are generated, project
 analytic account is automatically set.
+Add method mark_analytic_mrp_from_child for to mark the new field, analytic_mrp_from_child,
+to true the debit journal items from child productions.
```
Comment (dalonsod) [inline mrp_project_analytic/readme/DESCRIPTION.rst]: Better this:
```suggestion

Also adds a new check that identifies those debit journal items linked to MRP consume moves that are actually come from child productions.
```

**Review by dalonsod — APPROVED** [review]
:+1: please squash both commits in this case (we won't cherry-pick anything that could justify separate changes in two commits)

**Comment by Anxo82** [comment]
Review please

**Comment by dalonsod** [comment]
Let's merge!

**Pull request merged**